pilot/pkg/proxy/envoy/v2/gen2.go

```diff
@@ -116,6 +123,7 @@ func (s *DiscoveryServer) handleCustomGenerator(con *XdsConnection, req *xdsapi.
 	w.LastSent = time.Now()
 	w.LastSize = sz // just resource size - doesn't include header and types
 	w.NonceSent = resp.Nonce
+	adsLog.Infof("Pushed %s to %s count=%d size=%d", w.TypeUrl, con.node.ID, len(cl), sz)
```

Comment:
can you use consistent logging format with the rest of code? like `adsLog.Infof("ADS:CDS: REQ %v version:%s", con.ConID, discReq.VersionInfo)`